### PR TITLE
Server side datatables i36 fix

### PR DESCRIPTION
**Pull request by abubaker47**
Datatable added for vocabulary table and configured to work for other tables too.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
         "spatie/laravel-activitylog": "^3.2",
         "spatie/laravel-analytics": "^3.5",
         "unisharp/laravel-filemanager": "dev-db-support",
-        "facebook/graph-sdk" : "~5.0",
```
Comment (jamshidhashimi) [inline composer.json]: Why you removed `facebook/graph-sdk`? @abubaker47

Comment (abubaker47) [reply composer.json]: I have removed it based on our previous [conversation](https://github.com/ddlibrary/website/pull/42#pullrequestreview-247579160) regarding adding the facebook/graph-sdk which is automatically added by datatables plugin so I removed it.

Comment (jamshidhashimi) [reply composer.json]: @abubaker47 please do not remove that, as if I push this change, then the Facebook analytics won't work.

**Comment by jamshidhashimi** [comment]
Closing this, as it was decided that @abubaker47 will send specific PRs.

**Pull request closed without merging**